core/src/main/resources/hudson/Messages_de.properties

```diff
@@ -20,37 +20,37 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-FilePath.did_not_manage_to_validate_may_be_too_sl=Konnte {0} nicht \u00FCberpr\u00FCfen (vermutlich zu langsam)
-FilePath.validateAntFileMask.whitespaceSeprator=\
-  Leerzeichen k�nnen nicht mehr als Trennzeichen verwendet werden. Bitte verwenden Sie stattdessen '','' als Trennzeichen.
+FilePath.did_not_manage_to_validate_may_be_too_sl=Konnte \u201E{0}\u201C nicht \u00FCberpr\u00FCfen (vermutlich zu langsam)
```

Comment:
Please add guidelines for quoting to the wiki page: \u201E, \u201C for typographic double quotes, \u2018, \u2019 for typographic single quotes.